### ISSUE TITLE
Additional Design Considerations for Dark Mode

### ISSUE THREAD

**Issue by meepsterleapster** [wordplaydev/wordplay#383]
## What's the problem?

Browsers such as Google Chrome allow for pages that allow it to have their background colors replaced with the colors of the user's choice. The most common use case for these features is "Dark Mode" for users, but sometimes a variety of other recolors are used.

Wordplay was designed with light mode in mind, putting some text and features non-compliant with WCAG 2.2 guidelines when they are recolored. This makes certain pages hard to parse, confusing users and hurting accessibility for a subset of users.  

## What's the design idea?

Choices should be made in how Wordplay approaches browser alterations. Including whether or not supporting these alterations is a good idea in the first place. If we do choose to offer support, what forms this will take,  as well as how far we take that support are important questions to be asked.

## Who benefits?

The improvements to accessibility can help make wordplay more approachable to a wider audience .

## Design specification

(This section should be included after a design proposal is ready and approved, and the `buildable` tag is added. This text can remain until then. Designers should add their proposal here, not in a comment).

**Comment by meepsterleapster** [comment]
<img width="960" alt="image" src="https://github.com/wordplaydev/wordplay/assets/155786514/bfcd333f-00ee-4062-9bbd-c9a277941141">


An example of what dark mode wordplay looks like. Of specific note here is that the subtitles underneath the links are not contrasted enough with the now black background

**Comment by amyjko** [comment]
Thanks for reporting @meepsterleapster! Can you rewrite this as a defect, not an enhancement? As part of it, try to be precise about exactly what aspects of dark mode are violating WCAG 2.2, so whoever works on this issue has a clear guideline for what they're fixing. WCAG compliance is non-negotiable for Wordplay, so any deviation is a defect. As part of this, change the title to be precise about what standard(s) are being violated.

Thanks!

**Comment by meepsterleapster** [comment]
Got it, I will take a look tomorrow to confirm exactly what violations were found will submit a revised issue with the required changes.
